default.nix

```diff
@@ -167,9 +167,9 @@ let iosSupport = system == "x86_64-darwin";
             '';
           });
 
-          libiconv = super.libiconv.overrideAttrs (old: lib.optionalAttrs (self.stdenv.hostPlatform.useAndroidPrebuilt or false) {
-            configureFlags = [ "--disable-shared" "--enable-static" ];
-          });
+          #libiconv = super.libiconv.overrideAttrs (old: lib.optionalAttrs (self.stdenv.hostPlatform.useAndroidPrebuilt or false) {
+          #  configureFlags = [ "--disable-shared" "--enable-static" ];
+          #});
```

Comment:
Think we can remove this 
https://github.com/NixOS/nixpkgs/pull/355543/commits/76032f6107dd452ce03814807511c923f50806d4